src/components/Search/index.css

```
@@ -0,0 +1,34 @@
+:root {
+  --search-width: 100%;
+}
+
+.Search {
+  margin: 10px;
+  padding: 10px;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+
+  width: var(--search);
```

У тебя тут к несуществующей переменной идет обращение
```suggestion
  width: var(--search-width);
```